tests/nunit.runner.tests.iOS/AppDelegate.cs

```diff
@@ -53,6 +53,8 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
 
             // If you want to add tests in another assembly
             //nunit.AddTestAssembly(typeof(MyTests).Assembly);
+            // Or, if you want to add tests with an extra test options dictionary
+            //nunit.AddTestAssembly(typeof(MyTests).Assembly, new Dictionary<string, object>());
```

Comment:
Since these are our tests I would prefer if you pass in a parameter as an example and add a test to check that the the parameter was received properly. Just a simple `Assert.That(param, Is.Equal.To("Parameter")` type test will do. I just want to see this working end to end 😄

Reply:
Sorry that I have forgotten about this for a while 😔
There is one problem about adding extra tests for checking the passed-in parameters - all the tests are actually loaded automatically, without any parameters, when the test runner is first initialized. If I call nunit.AddTestAssembly here, it will load the test assembly again, now with extra parameters. Running the tests twice is okay, which is what uwp test runner's currently doing. However, for an individual test, it does not know whether a test run has parameters or not since it has to run once without any parameters no matter what.
On the other hand, if we only want to make sure that a parameter was received properly, I can add a test that passes when there's no parameter or when the parameter value matches. The only downside is that it won't catch the case where parameters are never passed through, but it can at least check the accuracy of the parameters. What do you think?